docs/solidity-by-example.rst

```diff
@@ -606,6 +606,7 @@ Safe Remote Purchase
         }
 
         /// Confirm the purchase as buyer.
+        /// By default state is set to create.
```

Comment:
This comment describes the semantics of this function. The fact that `state` is initially (and not by default) set to `State.Create` is not relevant for a user who wants to invoke this function.

Reply:
Left a comment on the original issue: https://github.com/ethereum/solidity/issues/3849#issuecomment-396665015